### PR TITLE
chore: add volunteer bruxx-6243

### PR DESCRIPTION
**Pull request by bruxx-6243**

### CODE REVIEW

**Comment by bruxx-6243** [comment]
@accodes21 You think that I should've merge my PR on the main branch instead of creating new branches ?

Because I realized that my profile is visible only on the **Preview** , and I'll like to correct some issues like

-Providing a default profile picture for those users who the src is not giving back the image

-and some styling issues like centering those profile user to be on the middle.

@accodes21 Try to see the responsive on less than **705px** 

image

**Comment by codewithvoid** [comment]
@bruxx-6243 Thanks for volunteering with AskDevs!

> You think that I should've merge my PR on the main branch instead of creating new branches ?
We don't have a defined git workflow atm. So new branch or PR from main branch both works.

>  why my profile is visible only on the preview not on the principal site?
It will be after being reviewed and merged in main repo.

Hope that helps.

File: data/users/bruxx-6243.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "Fariol Blondeau",
+    "bio": "Frontend Developer",
+    "image": "https://avatars.githubusercontent.com/u/81830567?v=4",
+    "categories": [
+        {
+            "slug": "web-development",
+            "topics": [
+                "css",
+                "javascript",
+                "Typescript",
```
Comment (codewithvoid) [inline data/users/bruxx-6243.json:11]: move programming languages to programming topics. just keep css and ReactJS here

**Comment by bruxx-6243** [comment]
I've done some mistakes

**Pull request closed without merging**